scripts/tex2markdown.py

```diff
@@ -328,6 +321,22 @@ def convert(ifile, ofile, title):
             label = x.group('label')
             data = data.replace(x.group(0),'<ex><label>'+label+'</label>')
 
+        # backspaces
+        data = data.replace("\\$", "$")
+        data = data.replace("\\#", "#")
+        data = data.replace("\\%", "%")
+        # data = data.replace(r"\\{}", " ")
+        data = data.replace(r'\{', '{')
+        data = data.replace(r'\}', '}')
+        data = data.replace("\\\\", " ")
+        data = data.replace("\\ ", "")
+        data = data.replace("etc.\\", "etc.")
+        data = data.replace("etc.)\\", "etc.)")
```

Comment:
The above two cases should be covered by the previous one (backslash+space).

Reply:
'etc.\\' and 'etc.)\\' do show up though. They might have a line break instead of a space.

Reply:
Yes! That is true.